packages/react/src/themes/teams-dark/colors.ts

```diff
@@ -291,3 +291,40 @@ export const colorScheme: ColorSchemeMapping = {
     background: colors.yellow[100],
   }),
 }
+
+const createCategoryColorScheme = (color: string, customValues = {}) => {
+  return {
+    foreground1: categoryColors[color][250],
```

Comment:
Let's replaced `foreground1` with `foreground` and `foreground2` with `foreground1`

Reply:
Also, is there a plan for the future, more of these colors to be added in the color scheme? Not sure why we have so many values in the color palette, but we clearly only use a subset of them..

Reply:
Moving the second comment discussion into the conversation with Sue Nguyen.